.github/workflows/build.yml

```diff
@@ -50,39 +50,56 @@ jobs:
         with:
              dockerfile: "Dockerfile"
 
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+        
       - name: Get Short SHA
         id: sha
         run: echo ::set-output name=short::$(echo $GITHUB_SHA | cut -c -7)
 
-      - name: Build only
-        uses: docker/build-push-action@v1
-        if: github.ref != 'refs/heads/master'
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+             ${{ runner.os }}-buildx-
+             
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1 
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PERSONAL_ACCESS_TOKEN }}
-          repository: ${{ env.DOCKERHUB_REPOSITORY }}
-          always_pull: true
-          add_git_labels: true
-          tag_with_ref: true
-          tag_with_sha: true
+          
+      - name: Build only
+        uses: docker/build-push-action@v2
+        if: github.ref != 'refs/heads/master'
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+          tags: |
+                 ${{env.DOCKERHUB_REPOSITORY}}:${{ steps.sha.outputs.short }}
```

Comment:
This changes the tag format from `docker/repository:sha-XXXXXXX` to `docker/repository:sha-XXXXXXXX` which I guess would cause issues in the QA and Production pipelines.

I think it needs to be

```
${{env.DOCKERHUB_REPOSITORY}}:sha-${{ steps.sha.outputs.short }}
```

Reply:
you are right, thank you